go.mod

```diff
@@ -2,24 +2,24 @@ module github.com/appuio/appuio-cloud-agent
 
 go 1.23
 
-toolchain go1.23.0
+toolchain go1.23.3
 
 require (
-	github.com/appuio/control-api v0.33.3
+	github.com/appuio/control-api v0.33.4
 	github.com/go-logr/logr v1.4.2
 	github.com/minio/pkg v1.7.5
-	github.com/openshift/api v0.0.0-20240729140855-0a58f8c30a8c // release-4.15
+	github.com/openshift/api v0.0.0-20241113165232-7ba4661b0aa5 // release-4.15
```

Comment:
Still release 4.15?

Reply:
This was `master` from `go get -u`. I've updated the PR to update the openshift api to release-4.16